test/common/ssl/test_data/unittest_expired_cert.cfg

```diff
@@ -0,0 +1,27 @@
+[ req ]
```

Comment:
Could you copy one of the existing files in this directory, perhaps `selfsigned_cert.cfg` to match the style?

Alternatively, you could just use `selfsigned_cert.cfg` when generating cert.